docs/modules/ROOT/pages/documentation/ecosystem/intellij-plugin/README.adoc

```diff
@@ -0,0 +1,32 @@
+= Payara IntelliJ IDEA Plugin
+
+Payara IntelliJ IDEA tools provide integration of the Payara Platform into IntelliJ IDEA.
+The extension is available in the https://plugins.jetbrains.com/plugin/15114-payara-tools[JetBrains Plugins Marketplace].
+
+* <<installing-plugin,Installing Payara IntelliJ plugin>>
+* <<tools,Payara Platform Tools>>
+
+[[installing-plugin]]
+== Installing Payara IntelliJ plugin
+
+1. You can install plugin from the main menu *File > Settings > Plugins dialog*
+
+2. In the Plugins dialog, switch to the Marketplace tab.
+
+3. In the dialog that opens, search for *Payara Tools*.
+
+4. Click the *Install* button:
+image::intellij-plugin/install-marketplace.png[alt=Visual Studio Marketplace, width="60%"]
```

Comment:
```suggestion
image:intellij-plugin/install-marketplace.png[alt=Visual Studio Marketplace, width="60%"]
```
This has to be an inline image, otherwise it won't be displayed (I tested it locally)